xdcr/rosmar_xdcr.go

```diff
@@ -111,6 +111,11 @@ func (r *rosmarManager) processEvent(ctx context.Context, event sgbucket.FeedEve
 			base.WarnfCtx(ctx, "Skipping replicating doc %s, could not perform a kv op get doc in toBucket: %s", event.Key, err)
 			r.errorCount.Add(1)
 			return false
+		} else if base.IsDocNotFoundError(err) {
+			// Log if the target document exists as tombstone but doesn't have xattrs we care about
+			if actualTargetCas != 0 {
+				base.DebugfCtx(ctx, base.KeyVV, "Target document exists as a tombstone, but does not have _vv, _mou, _sync")
+			}
```

Comment:
Is this useful to leave in debugging? At this point it might or might not get replicated depending on conflict resolution, despite having no _vv

Reply:
I left it in as it's a bit of an unusual case that I don't expect us to hit in future (when we've removed the need for the dummy xattr), and it would be useful to see when it continues to show up in that case.